src/utils/file-helper.ts

```diff
@@ -4,18 +4,64 @@ export namespace FileHelper {
 	/**
 	 * truncate file name according to wrapping element width
 	 */
-	export const truncateFileName = (fileName: string, widthOfElement: number) => {
-		// every 8px increment ~= 1 char width
-		// TODO: how to better determine char width?
-		const threshold = Math.floor(widthOfElement / 8);
-		const ellipsis = "...";
-		const length = fileName.length + ellipsis.length;
-		if (length > threshold) {
-			const thresholdIndex = threshold / 2;
-			return fileName.substring(0, thresholdIndex) + ellipsis + fileName.substring(length - thresholdIndex);
+	export const truncateFileName = (
+		fileName: string,
+		ref?: React.MutableRefObject<HTMLDivElement | HTMLParagraphElement>
+	) => {
+		let truncateFileName = fileName;
```

Comment:
avoid naming the variable the same as the function
```suggestion
		let truncatedFileName = fileName;
```

Reply:
updated